analytics/overview.md

```diff
@@ -67,17 +41,111 @@ Here are some real-world questions where metrics and dimensions from Analytics c
 - Short-form videos
     - *Have my viewers watched the first N videos on the feed until the end?* → `end`
     - *How many times have my viewers watched a specific video?* → `play-total` + `media-id`
+    - *What was the highest number of concurrent viewers last week?* → `from` + `ccv-peak` + `unique`
+    - *What videos have the longest view duration?* →  `view` + `media-id` + `sortBy[metricValue]` + `sortOrder=desc`
     
 - Advertising
     - *How many impressions have my ads generated?*  → `impression`
-    - *Is my ad displayed fast in Asia?*  → `impression-time` + `continent`
+    - *Is my ad displayed fast in Asia?* → `impression-time` + `continent`
+    - *From the websites where my video ad is embedded, which ones got the most views?* → `view` + `referrer` + `sortBy[metricValue]`
+
+### Requirements
 
 <Callout pad="2" type="info">
-**Testing**
 
-You can test the Analytics endpoints **in api.video's sandbox environment**. Check out [Environments](/reference#environments) for more details. The sandbox environment returns data from the last 24 hours.
+The Analytics features are available using api.video's video player. Check out the [Video Player SDK](/sdks/player/apivideo-player-sdk) for details about the implementation.
 </Callout>
 
+## How it works
+
+Analytics uses player events to analyze and segment your viewers' interactions with your content. Here are some key aspects:
+
+- By default, api.video retains analytics data for 30 days. You can extend data retention to 3 months or 12 months through the [Analytics page in the Dashboard](https://dashboard.api.video/analytics) - click on the `3M` or `12M` buttons to get started!
+- Player events are generated when your viewers engage with a video or live stream session.
+- Data is refreshed in real time, with a frequency of `<5s`.
+- Data does not carry over from the previous versions of Analytics, and unique viewer data starts only from `2024-08-14`.
+- `unique` viewers are only unique for a day.
+- Video re-plays using the dedicated re-play button in the player do not generate events.
+- On web, if a user watches a video, refreshes the browser tab, then plays the video again, a new event is generated.
+
+<Callout pad="2" type="success">
+api.video provides calculates the total number of plays through 2 dedicated endpoints:
+
+- `/data/metrics/play/total`
+- `/data/buckets/play-total/media-id`
+
+These 2 endpoints return data that is not limited to 30 days of retention, and can return data from earlier versions of Analytics.
+</Callout>
+
+### Sessions
+
+User sessions affect how data like the number of viewers is counted. 
+
+* For web users, a browser tab counts as one user session. 
+* For mobile users, a session starts when an app is launched. Apps going in the background remain in session. A new session starts when an app stops or is destroyed, and then launches again.
```

Comment:
`app stops or is destroyed` to be replaced with `app is destroyed`.

Reply:
Fixed ✅